content/getting-started/service-creation/components/index.md

```diff
@@ -6,57 +6,151 @@ weight: 100
+          <li>Title type (H2, H3, H4)</li>
+      </td>
+      <td><img alt="Title component - edit properties" src="props-title.png?width=700" title="Title component - edit properties" /></td>
+      <td><img alt="Title component - default look" src="display-title.png?width=700" title="Title component - default look" /></td>
+    </tr>
+    <tr>
+      <td>Paragraph</td>
+      <td>
+        <ul>
+          <li>Paragraph text</li>
+      </td>
+      <td><img alt="Paragraph component - edit properties" src="props-paragraph.png?width=700" title="Paragraph component - edit properties"/></td>
+      <td><img alt="Paragraph component - default look" src="display-paragraph.png?width=700" title="Paragraph component - default look" /></td>
+    </tr>
+  </tbody>
+</table>
 
 ### Basic form components
 
-#### Input
-##### Default look
-{{<figure src="display-input.png?width=1000" title="Input component - default look">}}
-
-##### Edit properties
-{{<figure src="props-input.png?width=1000" title="Input component - edit properties">}}
-
-#### Checkboxes
-##### Default look
-{{<figure src="display-checkbox.png?width=1000" title="Checkboxes component - default look">}}
-
-##### Edit properties
-{{<figure src="props-checkbox.png?width=1000" title="Checkboxes component - edit properties">}}
-
-#### Radio buttons
-##### Default look
-{{<figure src="display-radio.png?width=1000" title="Radio buttons component - default look">}}
-
-##### Edit properties
-{{<figure src="props-radio.png?width=1000" title="Radio buttons component - edit properties">}}
-
-#### Dropdown
-This component is under development. The documentation will be updated with more details as the component is updated.
-
-#### Text area
-This component is under development. The documentation will be updated with more details as the component is updated.
-
-#### File attachment
-This component is under development. The documentation will be updated with more details as the component is updated.
-
-#### Container
-This component is under development. The documentation will be updated with more details as the component is updated.
+<table>
+  <tbody>
+    <tr>
+      <th>Component name</th>
+      <th>Available properties</th>
+      <th>Properties editor</th>
+      <th>Default look</th>
+    </tr>
+    <tr>
+      <td>Input</td>
+      <td>
+        <ul>
+          <li>Link to data model</li>
+          <li>Label text</li>
+          <li>Description text</li>
+      </td>
+      <td><img alt="Input component - edit properties" src="props-input.png?width=700" title="Input component - edit properties" /></td>
+      <td><img alt="Input component - default look" src="display-input.png?width=700" title="Input component - default look" /></td>
+    </tr>
+    <tr>
+      <td>Checkboxes</td>
+      <td>
+        <ul>
+          <li>Link to data model</li>
+          <li>Label text</li>
+          <li>Description text</li>
+          <li>Method to add options (codelist/manual)</li>
```

Comment:
Should there maybe be two images under "properties editor"? Showing for codelist as well as manual?

Reply:
this applies to other components with this choice as well

Reply:
Good idea, updated with two images, one for manual and one for codelist. Updated for both radiobuttons and checkboxes.